src/screens/Main.tsx

```diff
@@ -34,14 +34,67 @@ interface Props {
   className?: string;
 }
 
+interface MenuActionItemsProps {
+  idx: number;
+  title: string;
+  isEnabled: boolean;
+  component: React.ReactElement;
+}
+
+const MenuContents = [
+  {
+    idx: 0,
+    title: 'Transfer',
+    isEnabled: true,
+    component: (
+      <>
+        <Transfer />
+        <ExtensionAccountCheck component={<Accounts />} />
+      </>
+    )
+  },
+  {
+    idx: 1,
+    title: 'Remark',
+    isEnabled: true,
+    component: <Remark />
+  },
+  {
+    idx: 2,
+    title: 'Custom Call',
+    isEnabled: true,
+    component: <CustomCall />
+  },
+  {
+    idx: 3,
+    title: 'Connect to a wallet',
+    isEnabled: true,
+    component: <p>Connect to a wallet</p>
```

Comment:
What is this one supposed to do?

Reply:
This will appear when no extension is attached:
https://www.figma.com/file/OqRXj7Th4uZhSWjNQn8CmH/Bridge-UI?node-id=749%3A121

For the context of this PR I just let the placeholder as was before (will be implemented in different PR)